shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -62,6 +62,30 @@ async fn test_passthrough(#[case] driver: CassandraDriver) {
     batch_statements::test(&connection).await;
 }
 
+#[cfg(feature = "cassandra-cpp-driver-tests")]
+#[rstest]
+#[cfg_attr(feature = "cassandra-cpp-driver-tests", case(Datastax))]
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn test_passthrough_encode(#[case] driver: CassandraDriver) {
```

Comment:
Can't we just put another transform chain on the passthrough one and run the tests against that. Save starting another docker compose.

Reply:
Currently the test suite does not clean up after itself so that wouldnt work.